private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -104,13 +117,13 @@ class Designer extends React.Component<DesignerProps> {
 
   render() {
     return (
-      <DesignerWrapper id="designer-container" dangerouslySetInnerHTML={{ __html: "dangerouslySetACDesigner" }}></DesignerWrapper>
+      <React.Fragment>
+        <DesignerWrapper id="designer-container" dangerouslySetInnerHTML={{ __html: "dangerouslySetACDesigner" }}></DesignerWrapper>
```

Comment:
I know this is not part of your PR but change this to just 
`<DesignerWrapper id="designer-container" />`